Makefile

```diff
@@ -7,6 +11,18 @@ build-tinygo: examples/filter-simple/main.wasm examples/noop/main.wasm
 %/main-debug.wasm: %/main.go
 	@(cd $(@D); tinygo build -o main-debug.wasm -scheduler=none -target=wasi main.go)
 
+.PHONY: testdata
+testdata:
+	@$(MAKE) build-tinygo
+	@$(MAKE) build-wat
+
+.PHONY: build-wat
+build-wat: $(wildcard scheduler/test/testdata/*/*.wat)
```

Comment:
this makefile change will conflict with #25, but I will handle it later. To me it doesn't matter which is merged first, but we should try to merge one or the other (or both) today because the build is currently broken.